pubspec.yaml

```diff
@@ -109,51 +107,45 @@ dependencies:
   flutter_dotenv: ^5.1.0
 
   # Notifications & Logging
-  flutter_local_notifications: ^15.1.0+1
+  flutter_local_notifications: ^17.1.2
   logger: ^2.0.1
 
   # Error handling
-  device_info_plus: ^9.0.3
+  device_info_plus: ^10.1.0
 #  flutter_mailer: ^2.0.0
 #  fluttertoast: ^8.2.2
 
   # Package information
-  package_info_plus: ^4.1.0
+  package_info_plus: ^8.0.0
 
   # Path, permission & Markdown handling
   path_provider: ^2.1.0
-  permission_handler: ^10.4.3
-  flutter_markdown: ^0.6.17+1
+  permission_handler: ^11.3.1
+  flutter_markdown: ^0.7.2+1
 
   # Ads
   google_mobile_ads: ^5.0.0
   retry: ^3.1.2
   # Generate bindings to native libraries
   ffi: ^2.1.0
   # Deeplink handling
-  app_links: ^3.5.1
+  app_links: ^6.1.1
   #Loading
   animated_loading_border: ^0.0.2
   shimmer: ^3.0.0
 
-
-# wakelock ^0.6.2 requires win32 ^2.0.0 or ^3.0.0
-# See https://github.com/creativecreatorormaybenot/wakelock/issues/211
-dependency_overrides:
-  win32: 3.1.4
```

Comment:
I don't think we need this since we aren't using [wakelock](https://pub.dev/packages/wakelock) anymore